setup.py

```diff
@@ -124,8 +132,7 @@ def check_python_version():
 
 if __name__ == "__main__":
     check_python_version()
```

Comment:
is there a reason why we don't simply use the `python_requires` kwarg? See https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords

Reply:
None that I know of, but this is unrelated to this PR, which is focused on making the minimal changes necessary to avoid package imports. Please could you open an issue (or a PR)?